SU-SWS/stanford_basic
# D8CORE-1022 Tweak Edit Buttons display in D8 page authoring

# READY FOR REVIEW

# Summary
- Requested D8 page authoring edit button tweaks. The style should **only** appear when the user is logged on.

# Review By (Date)
- asap

# Urgency
- Low (`v1.1.0`)

# Steps to Test
1. Check out this branch
2. Run `npm run publish`
3. Login to local
4. View any page with authoring options.

# Affected Projects or Products
- Stanford Basic

# Associated Issues and/or People
- [D8CORE-1022](https://stanfordits.atlassian.net/browse/D8CORE-1022)
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
